test/quicfaultstest.c

```diff
@@ -315,16 +316,9 @@ static int test_corrupted_data(int idx)
      * Introduce a small delay so that the above packet has time to be detected
      * as lost. Loss detection times are based on RTT which should be very
      * fast for us since there isn't really a network. The loss delay timer is
-     * always at least 1ms though. We sleep for 100ms.
-     * TODO(QUIC): This assumes the calculated RTT will always be way less than
-     * 100ms - which it should be...but can we always guarantee this? An
-     * alternative might be to put in our own ossl_time_now() implementation for
-     * these tests and control the timer as part of the test. This approach has
-     * the added advantage that the test will behave reliably when run in a
-     * debugger. Without it may get unreliable debugging results. This would
-     * require some significant refactoring of the ssl/quic code though.
+     * always at least 1ms though. We skip forward 100ms
      */
-    OSSL_sleep(100);
+    qtest_add_time(100);
```

Comment:
What was the problem with OSSL_sleep()? Isn't this just working around a real problem that we should solve somehow?

Reply:
This could fail on some systems non-deterministically probably due to the assumptions in the test about RTT being incorrect. I could probably solve it by increasing the sleep time, but by swapping to fake time we make everything deterministic and we know it should work everywhere. I don't think there is a real problem here at all.

Reply:
OK